CHANGELOG.md

```diff
@@ -1,5 +1,5 @@
 # Changelog
- 
+
```

Comment:
How come these `*.md` files are affected? It seems to me that they shouldn't be.